src/main.js

```diff
@@ -5,53 +5,20 @@ let path               = require('path');
 let debug              = require('debug')('server');
 let Promise            = require('promise');
 let AWS                = require('aws-sdk-promise');
-let raven              = require('raven');
 let exchanges          = require('./exchanges');
 let ScopeResolver      = require('./scoperesolver');
 let signaturevalidator = require('./signaturevalidator');
 let taskcluster        = require('taskcluster-client');
 let url                = require('url');
-let validate           = require('taskcluster-lib-validate');
-let loader             = require('taskcluster-lib-loader');
-let app                = require('taskcluster-lib-app');
 let SentryManager      = require('./sentrymanager');
 
 // Create component loader
-let load = loader({
+let load = base.loader({
```

Comment:
Why go back to using tc-base?  The move to use the individual libs was deliberate :)